### PR TITLE
PB-8480: Changes for px-backup 2.7.3 release

### PR DESCRIPTION
**Pull request by dbinnal-px**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: px-central/values.yaml [hunk 1]
```diff
@@ -166,33 +189,75 @@ images:
...
+    tag: 2.7.0
   keycloakInitContainerImage:
     registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
     repo: portworx/library
     imageName: busybox
-    tag: 1.31
+    tag: 1.35.0
   mysqlImage:
     registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
     repo: portworx/library
```
Comment (ss-px) [inline px-central/values.yaml:205]: @dbinnal-px Is the repo different ie `portworx/library` only for this image?

File: px-central/values.yaml [hunk 2]
```diff
@@ -166,33 +189,75 @@ images:
...
+  pxBackupPrometheusConfigReloaderImage:
+    registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
+    repo: portworx
+    imageName: prometheus-config-reloader
+    tag: v0.75.0
 
+  ## Images required for intermediate Mongo upgrades
+  mongodbImageMap:
+    registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
+    repo: portworx
```
Comment (ss-px) [inline px-central/values.yaml:258]: @dbinnal-px Will the repo for this also have to be `portworx/bitnami` ?

Comment (dbinnal-px) [reply px-central/values.yaml:258]: Yes, it was followed

**Review by ss-px — APPROVED** [review]

File: stable/index.yaml [hunk 3]
```diff
@@ -25,11 +25,11 @@ entries:
     sources:
     - https://github.com/portworx/aws-helm
     urls:
-    - https://raw.githubusercontent.com/portworx/aws-helm/master/stable/portworx-1.1.4.tgz
+    - https://raw.githubusercontent.com/portworx/aws-helm/2.7.3/stable/portworx-1.1.4.tgz
```
Comment (vikasit12) [inline stable/index.yaml:28]: update the link back to master

**Review by vikasit12 — CHANGES_REQUESTED** [review]

File: stable/index.yaml (hunk 3 above)
Comment (dbinnal-px) [reply stable/index.yaml:28]: Sure, I will change it before merging

**Review by vikasit12 — APPROVED** [review]

**Pull request merged**